includes/class-admin.php

```diff
@@ -167,12 +167,58 @@ function bu_liaison_inquiry_options_page_html() {
 		// If there is already a key set, use it to fetch and display a field inventory.
 		$options = get_option( 'bu_liaison_inquiry_options' );
 		if ( ! empty( $options['APIKey'] ) ) {
+			$api = new Spectrum_API( null, $options['APIKey'] );
 		?>
+		<h2>Select Liaison Form</h2>
+
```

Comment:
Would maybe adding a hr above this help distinguish that this is not part of the form above?

Also maybe add a line below it explaining why they should select a form? or maybe set the content of the h2 to something like: `Preview shortcode and field inventory`.

This is just me nitpicking and not really important, feel free to disregard it if you think the effort is not worth it.
I just thought it could use a little more help making it more intuitive but like I said, not really important.